src/test/java/uk/gov/hmcts/ccd/endpoint/ui/QueryEndpointIT.java

```diff
@@ -1199,6 +1201,42 @@ public void getEventTriggerForCaseType_valid() throws Exception {
         assertThat(field2.getDisplayContext(), equalTo("OPTIONAL"));
     }
 
+    @Test
+    public void getEventTriggerForCaseType_shouldAddFieldsWithCERATORCaseRole() throws Exception {
```

Comment:
I think it would be good to create a new test class dedicated to Case Roles rather than adding it to the QueryEndpointIT one? For example `CaseRolesIT`?